DominicGBauer/system-design-dashboard
# feat: add line graph

## Scope
https://dev.azure.com/BRXDOM001/Dream%20Team%20Dashboardin/_workitems/edit/107

## Work Done
1. Replaced Piehcart with LineGraph
2. Updated tests

## Steps to Test
1. Pull the branch
2. Run the app
3. Make sure you see the line graph

## Screenshot
![image](https://user-images.githubusercontent.com/46312751/119100570-6025bf00-ba18-11eb-90ac-7c66fa0a1ba8.png)
